tests/acceptance/test_project_user_feedback.py

```diff
@@ -12,7 +12,10 @@ def setUp(self):
         self.org = self.create_organization(
             owner=self.user, name='Rowdy Tiger')
         self.team = self.create_team(
-            organization=self.org, name='Mariachi Band')
+            organization=self.org,
+            name='Mariachi Band',
+            members=[self.user]
```

Comment:
It was necessary to assign the user to the team to properly generate the correct screenshots.